common/tools/turborepo-remote-cache/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Use an official Ubuntu base image
+FROM ubuntu:24.04
```

Comment:
would some smaller nodejs images work too? like alpine linux based ones

Reply:
https://hub.docker.com/_/node/

Reply:
Ok, will try in a follow on PR